code/controllers/subsystem/shuttles/supply.dm

```diff
@@ -195,6 +195,10 @@
 				return 1
 			if(istype(A,/obj/item/device/radio/beacon))
 				return 1
+			if(istype(A,/obj/machinery/the_singularitygen/))
```

Comment:
there's enough items here now for it to be cleaner to replace it with a blacklist and do is_type_in_list()